### ISSUE TITLE
Suggestion for increasing the coverage and accuracy of xrefs

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/MAxO#231]
From meeting with @drseb and me.

Some classes in MAxO like [Chest CT](https://www.ebi.ac.uk/ols/ontologies/maxo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMAXO_0010317) do not have an xref. Other classes, such as drug treatments ("[antipsychotic agent therapy](https://www.ebi.ac.uk/ols/ontologies/maxo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMAXO_0000166)") have xrefs to CHEBI - is this really useful? Wouldn't it be better to xref to the corresponding NCIT terms?

In general, we should try to use symbiont or some other approach to keep xrefs/mappings up to date and provide coverage across all classes in the hierarchy. I would personally like to move from xrefs to a more skos like approach to capture external mappings as well.

**Comment by LCCarmody** [comment]
I will need more clarity on what symbiont and skos are, but I'm definitely interested in improving where I can.

**Comment by pnrobinson** [comment]
NCIT has a term for everything, but its hierarchy is not always optimal for items that are outside of cancer types, and thus it would be difficult to actually use NCIT medication xrefs. In some future tooling, we would like to use the medication xrefs to get instances of all of the particular types of each of the chebi classes, and that seems to be the best option for that. 
Why are you suggesting using NCIT?

**Comment by LCCarmody** [comment]
FTR, we do use CHEBI roles in building the logical def, so that would be a way to get medication links to the terms.

**Comment by matentzn** [comment]
@pnrobinson yeah we dont really want to deal with NCIt, but we should give users of Maxo a mapping to go back and forth between NCIt and MAxO. But this ticket was more a general appeal to add rich mappings to all sorts of sources - a process I will support of course. The Chebi links should probably be maintained as logical axioms?

@LCCarmody do not worry, this ticket does not require an action yet - if you want, bring it up at our next meeting, then I can explain you the background a bit better!

**Comment by mellybelly** [comment]
we have been trying to improve representation in terms of OBO axioms in NCIt-plus,  you can view here  https://ols.ccdh.io/index

**Comment by matentzn** [comment]
Very interesting @mellybelly - is there a description of what NCIt-plus is over NCIt?

**Comment by mellybelly** [comment]
https://github.com/NCI-Thesaurus/thesaurus-obo-edition/wiki
https://yasgui.triply.cc/#
